H2TauTau/python/proto/analyzers/H2TauTauTreeProducerBase.py

```diff
@@ -115,13 +115,13 @@ def fillDiLepton(self, tree, diLepton):
     # lepton
     def bookLepton(self, tree, p_name):
         self.bookParticle(tree, p_name)
-        self.bookParticle(tree, p_name + '_jet')
+        self.bookJet(tree, p_name + '_jet')
```

Comment:
See other comment - this will increase the size of all our final trees so I would try to avoid filling this by default